cmd/k8s-operator/operator_test.go

```diff
@@ -1388,7 +1388,7 @@ func TestTailscaledConfigfileHash(t *testing.T) {
 		parentType:      "svc",
 		hostname:        "default-test",
 		clusterTargetIP: "10.20.30.40",
-		confFileHash:    "e09bededa0379920141cbd0b0dbdf9b8b66545877f9e8397423f5ce3e1ba439e",
+		confFileHash:    "362360188dac62bca8013c8134929fed8efd84b1f410c00873d14a05709b5647",
```

Comment:
nit: Maybe in future these tests could be improved a bit by using the `modifier` func to simply check that the hash is not the same as the previous one? I think that's the intention of including the hash (i.e. we don't care about any single value, only that it changes), but it causes a bit of unnecessary toil when the format etc gets updated. Probably not in this PR though